client/src/components/Profile.js

```diff
@@ -1,29 +1,147 @@
 import styled from "styled-components/macro";
```

Comment:
Maybe I'm  mistaken but shouldn't you also `import React from 'react' ` ?

Reply:
don't need it anymore :)